docs/support_playbook.md

```diff
@@ -104,6 +104,13 @@ ApplicationForm.find(_id).update!(becoming_a_teacher: 'new text', subject_knowle
 
 This is possible via the support UI.
 
+### Changing a deferred applicant's course in the new cycle
+
+If the deferred applicant would like to change their course and the provider accepts:
```

Comment:
Could we add a note around this being required as currently in manage/API confirming a deferred application in the new cycle doesn't allow changing a course so has to be done via support.

Could we also add that the course option being set should be in the previous cycle so the provider can confirm the deferred option themselves. 

If the course doesn't exist previously we'll need them to confirm the offer first, then we can change the course then to the new course in the current cycle.

Reply:
Thanks, I've added your suggestions and removed the other scenario 👍